coalib/coala.py

```diff
@@ -37,7 +37,8 @@ def main():
 
         console_printer = ConsolePrinter(print_colored=not args.no_color)
 
-        if args.json:  # needs to be checked in order to display bears in json
+        # needs to be checked in order to display bears in json
+        if args.json and not args.format:
```

Comment:
I can't agree with you on this:) I checked it, when I use `coala --format -- json` both of them are true